scripts/clear_submissions_from_queue.py

```diff
@@ -1,13 +1,14 @@
 from datetime import datetime, timedelta
 
 from praw import Reddit
-from praw.exceptions import RedditAPIException
+# from praw.exceptions import RedditAPIException
```

Comment:
Better not to comment code

Reply:
This is a throwaway script that's supposed to be modified in place depending on whatever we need -- if this is the only issue you have with the PR, then I would consider it good to go

Reply:
^^^ This.

Main reason I commented it out here instead of deleting is because some seemingly important try-except code was commented out later in the file, where it handles this particular exception. Rather than delete the commented code in its entirety, this file seems more like a living workspace just in case we need it. Might even be able to delete it if we rework it into one of the commands in the bot. 🤷